### ISSUE TITLE
no option for empty date

### ISSUE THREAD

**Issue by sandeshsapkota** [cogor/vue-formulate-datepicker#27]
Hi thank you for the good wrapper.

I am having a issue here with it. I am working on a form there Date of birth is required. 

but the vue formulate wrapper showing the default date `10 Jan 1970`.  

the problem is while  submitting the form, it has already data and it shows no error and it may cause that the use may miss to add their date of birth . 

Here is the code : 
```  
<FormulateInput
                type="datepicker"
                v-model="form.dob"
                typed-date="''"
              >
              </FormulateInput>
```

Thank you .

**Comment by cogor** [comment]
Hi, I'll fix it as soon as possible

**Comment by cogor** [comment]
@sandeshsapkota Fixed in v0.1.17
